<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-B5LLFCD0" failures="1" tests="6" name="Basic_TestNG.TestNG_priority_enabled" time="0.028" errors="0" timestamp="2021-11-12T13:24:53 IST" skipped="2">
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="RegisterUser" time="0.020">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at Basic_TestNG.TestNG_priority_enabled.RegisterUser(TestNG_priority_enabled.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- RegisterUser -->
  <system-out/>
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="ValidLogin" time="0.003"/>
  <system-out/>
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="HomePage" time="0.002"/>
  <system-out/>
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="AddBenificiary" time="0.003"/>
  <system-out/>
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="ValidateBenificiary" time="0.000">
    <skipped/>
  </testcase> <!-- ValidateBenificiary -->
  <system-out/>
  <testcase classname="Basic_TestNG.TestNG_priority_enabled" name="Logout" time="0.000">
    <skipped/>
  </testcase> <!-- Logout -->
  <system-out/>
</testsuite> <!-- Basic_TestNG.TestNG_priority_enabled -->
